conclave-host/src/main/kotlin/com/r3/conclave/host/internal/gramine/GramineEnclaveHandle.kt

```diff
@@ -65,6 +65,7 @@ class GramineEnclaveHandle(
         val command = mutableListOf(getGramineExecutable(enclaveMode))
         command += listOf(
             "java",
+            "-XX:-UseCompressedClassPointers",
```

Comment:
Reference https://r3-cev.atlassian.net/browse/CON-1274 and reopen it. We need to understand why do have to do this.

Reply:
I reopened the original ticket https://r3-cev.atlassian.net/browse/CON-1165 as CON-1274 was duplicating it.
Will add the reference in the code to CON-1165.